easybuild/easyconfigs/l/libev/libev-4.33-GCC-11.2.0.eb

```diff
@@ -0,0 +1,37 @@
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'ConfigureMake'
+
+name = 'libev'
+version = '4.33'
+
+homepage = 'http://software.schmorp.de/pkg/libev.html'
+description = """A full-featured and high-performance (see benchmark) 
+event loop that is loosely modelled after libevent, but without its 
+limitations and bugs. It is used in GNU Virtual Private Ethernet, 
+rxvt-unicode, auditd, the Deliantra MORPG Server and Client, and many 
+other programs."""
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://dist.schmorp.de/libev']
+sources = ['%(name)s-%(version)s.tar.gz']
+
+checksums = [
+    '507eb7b8d1015fbec5b935f34ebed15bf346bed04a11ab82b8eee848c4205aea',  # libev-4.33.tar.gz
+]
+
+builddependencies = [
+    ('binutils', '2.37'),
+    ('Autotools', '20210726'),
+]
+
+parallel = 1
```

Comment:
Is parallel builds really a problem here? That's usually a fortran thing. I tried without and it seemed OK.

Reply:
The idea was more: as it is building so fast, if you are having say 20 cores per default, does it make sense to try and build it on 20 cores when it is done at the same time, or less, by one core. So not a problem as such, more thinking of resources. Let me know what you think about that and I am happy to remove it.

Reply:
is this parameter affecting job-submissions as well? I never actually used it myself

Reply:
From what I understand, if you got say `EASYBUILD_PARALLEL="24"` as an environment variable, and thus in your job submission, you could build all your software with 24 cores apart from the ones which have `parallel = 1` in the EC file. At least that is what I observed when I installed software manually on our cluster.

Reply:
If parallel builds aren't a problem then don't use parallel = 1. Make handles that quite nicely.

Reply:
No problems. I have removed it.